spec/site/alarm_spec.rb

```diff
@@ -67,4 +67,43 @@ def check_scripts
   #    system(SCRIPT_PATHS['deactivate_alarm'])
   #  end
   #end
+
+  it 'Disruption test', :script, sxl: '>=1.0.7' do |example|
+    check_scripts
+    component = COMPONENT_CONFIG['detector_logic'].keys.first
+
+    TestSite.isolated do |task,supervisor,site|
+      site.log "Checking connection", level: :test
+      # Is an explicit check needed?
+    end
+      
+    # Deactivate alarm 
+    system(SCRIPT_PATHS['activate_alarm'])
```

Comment:
The comment says "deactivate alarm", but the script says "activate alarm"?

Reply:
Lazy copypasting on my part, my bad.